src/components/Map/layers/clusterLayer/index.js

```diff
@@ -144,7 +144,7 @@ export default class PostClusterLayer extends CompositeLayer {
 
           return d.properties.type + '.png'
         },
-        getSize: d => d.properties.cluster ? 50 : 22,
+        getSize: d => d.properties.cluster ? 34 : 15,
```

Comment:
Are these little styling tweaks, while you were looking at location stuff?

Reply:
yeah, unrelated. https://github.com/Hylozoic/hylo-evo/issues/956